setup/air-gapped.md

```diff
@@ -63,7 +67,7 @@ platform images are hosted in Coder's Docker Hub repo.
     docker push my-registry.com/coderenvs/coder-service:<version>
     ```
 
-1. Once all of the resources are in your air-gapped network, run the following
+3. Once all of the resources are in your air-gapped network, run the following
    to deploy Coder to your Kubernetes cluster:
 
     ```bash
```

Comment:
The following helm command is wrong.

- envmetrics is no more
- dockerd is no more
- missing dashboard
- missing envbox